### PR TITLE
Do not forward $SYS topics for +/# subscriptions.

### PR DESCRIPTION
**Pull request by mcollina**
Fixes: https://github.com/mcollina/aedes/issues/135

@ckrey

### CODE REVIEW

File: aedes.js [hunk 1]
```diff
@@ -200,8 +200,12 @@ function DoEnqueues () {
   }
 }
 
+// + is 43
+// # is 35
 function removeSharp (sub) {
-  return sub.topic !== '#'
+  var code = sub.topic.charCodeAt(0)
+  console.log(sub.topic, code)
```
Comment (GavinDmello) [inline aedes.js]: remove console.log

**Comment by GavinDmello** [comment]
This may affect the <a>https://github.com/mcollina/aedes/issues/123</a> change we were planning on implementing

**Comment by mcollina** [comment]
@GavinDmello yes. Better to have the tests in.

File: aedes.js (hunk 1 above)
Comment (mcollina) [inline aedes.js]: good catch!

Comment (mcollina) [inline aedes.js]: updated

**Comment by ckrey** [comment]
✅

**Pull request merged**